### PR TITLE
Add folium integration and use it in an example notebook

### PR DESCRIPTION
**Pull request by SamLau95**
With the discovery of the [folium](https://github.com/python-visualization/folium) package, I remove the old hacky stuff
I put in before for gmaps and use folium instead!

I also added an example notebook in the `examples/` folder that
successfully renders a map centered at Broad Street (now Broadwick
Street).

The next step is to add the points / polygon functionality, and we can call #5 a wrap.

### CODE REVIEW

**Comment by SamLau95** [comment]
screenshot 2015-07-21 23 00 23

File: datascience/maps.py [hunk 1]
```diff
@@ -19,8 +14,27 @@ def draw_map(center, zoom, points=[], regions=[]):
     zoom -- zoom level
     points -- a sequence of MapPoints
     regions -- a sequence of MapRegions
+
+    TODO(sam): Add points and regions functionality
     """
-    # Some magic to get javascript to display the map
+    map = folium.Map(
+        location=center,
+        tiles='Stamen Toner',
```
Comment (SamLau95) [inline datascience/maps.py]: Note that I chose this arbitrarily — if you feel like there's a better map style to use let me know.

**Comment by SamLau95** [comment]
Also — I didn't increment the package version. Do we increment the version on every PR?

**Comment by alvinwan** [comment]
That's fine - we only increment when we republish the package.

This feature would definitely warrant a new version, but we don't know how many other deployments we might make before this is ready.

**Pull request merged**